internal/driver/sqlite_base.go

```
@@ -0,0 +1,281 @@
+	if sb.async {
+		awaitKw, iteratorType, nextDef, iterDunder, nextDunder, stopExc, article = "await ", "AsyncIterator", "async def __anext__", "__aiter__", "__anext__", "StopAsyncIteration", "an "
+	}
+
+	body.QueryResults.WriteQueryResultsClassHeader(sb.ConnType(), []string{
+		fmt.Sprintf("self._cursor: %s | None = None", cursorType),
+		fmt.Sprintf("self._iterator: collections.abc.%s[%s] | None = None", iteratorType, sqliteResultType),
+	}, sqliteResultType, sb.async)
+	if sb.async {
+		body.QueryResults.WriteQueryResultsAwaitFunction([]string{
+			"result = await (await self._conn.execute(self._sql, self._args)).fetchall()",
+			"return [self._decode_hook(row) for row in result]",
```

golangci-lint (2.12.2)

[error] string `return [self._decode_hook(row) for row in result]` has 3 occurrences, make it a constant

(goconst)